web/src/components/base/ChipButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+
+import { CloseIcon } from '../../assets'
+import Icon from './Icon'
+
+const StyledButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  height: 30px;
+  padding: 4px 8px;
+  align-items: center;
+  margin: 0 2px;
+  border-radius: 20px;
+  gap: 4px;
+  background-color: ${props => props.theme.colors.backgroundColor};
+  color: ${props => props.theme.colors.textSecondaryColor};
+  font-family: ${props => props.theme.fonts.native.contentFontBold};
```

Comment:
this is a native font family and won't work. Please use web

Reply:
Replaced with 0.875 rem. At the moment we use pixel nearly everywhere though, so this should be thoroughly fixed with #2354.